src/identity/types.ts

```diff
@@ -179,6 +180,22 @@ export type UseAttestations = {
   schemaId: Address | null;
 };
 
+/**
+ * Note: exported as public Type
```

Comment:
When you write exported as public, make sure to export it in the `index` file as well :)

Reply:
done